e2e/collections_test/test_avg_different_types/test.yml

```diff
@@ -0,0 +1,5 @@
+query: >
+  CALL collections.avg([-5, 5, 2.2, -8.6, 0, 1.0])
+  YIELD average;
+output:
+  - average: -0.8999999999999999
```

Comment:
The empty line is missing here too and on some other files. Add empty line please on other files to